delfin/drivers/dell_emc/vmax/client.py

```diff
@@ -12,24 +12,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import PyU4V
-
 from oslo_log import log
 from oslo_utils import units
 
 from delfin import exception
 from delfin.common import constants
+from delfin.drivers.dell_emc.vmax import rest
 
 LOG = log.getLogger(__name__)
 
-SUPPORTED_VERSION = '90'
+EMBEDDED_SYMMETRIX_ARRAY_NUM = 1
```

Comment:
It is not Embedded symmetrix, but embedded unisphere . 
This will confuse developer, unless we provide a description . we can probably name EMBEDDED_UNISPHERE_ARRAY_COUNT